le.sh

```diff
@@ -279,6 +279,9 @@ _initpath() {
   if [ -z "$CA_CERT_PATH" ] ; then
     CA_CERT_PATH="$WORKING_DIR/$domain/ca.cer"
   fi
+  if [ -z "$CERT_CHAIN_PATH" ] ; then
```

Comment:
You're right on this one.

Also you can specify the path twice, or can be changed to do so. With that said, I somehow like that both files are separate, but you also have a full chain of it (Let's say you automate this for multiple systems), some servers needs a chain (haproxy e.g) - where others you specify the certificate and the CA separately.